clevertap-core/src/main/java/com/clevertap/android/sdk/InAppNotificationActivity.java

```diff
@@ -203,49 +205,75 @@ private void notifyInAppDismissed() {
         invokedInAppDismissCallback = true;
     }
 
+    @Nullable
     @Override
-    public void inAppNotificationDidClick(CTInAppNotification inAppNotification, Bundle formData,
-            HashMap<String, String> keyValueMap) {
-        didClick(formData, keyValueMap);
+    public Bundle inAppNotificationDidClick(
```

Comment:
why do we need to return a bundle here?

Reply:
To keep the behavior of didClick/didDismiss the same. Previously didClick was handled in CTInAppBaseFragment which triggered both didClick and didDismiss with the same data bundle. I have moved the data bundle handling in InAppController, where it could be reused. didClick now returns the data bundle and the caller of didDismiss can than pass the same bundle.